python/lsst/afw/cameraGeom/utils.py

```diff
@@ -25,8 +25,7 @@
 """
 Support for cameraGeom
 """
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
```

Comment:
sense you already have commits on this ticket for fixing formatting issues, this change should be moved to one of those commits.